root/static/scripts/account/components/EditProfileForm.js

```diff
@@ -139,8 +139,15 @@ class EditProfileForm extends React.Component<Props, State> {
     e: SyntheticEvent<HTMLSelectElement>,
     languageIndex: number,
   ) {
-    // $FlowFixMe ~ string incompatible with FluencyT's string literals
-    const selectedFluency: FluencyT = e.currentTarget.value;
+    const selectedValue = e.currentTarget.value;
+    let selectedFluency: FluencyT | null = null;
+    switch (selectedValue) {
+      case 'basic':
+      case 'intermediate':
+      case 'advanced':
+      case 'native':
+        selectedFluency = selectedValue;
+    }
```

Comment:
This error wasn't wrong TBH, since the string we get from the HTML could be anything (for example, we could introduce a bug where the `value` attributes are incorrect). But maybe that's unlikely and the `switch` statement is overkill.

Reply:
A better solution might be to just change the type to `FieldT<string | null>,`.

Reply:
I don't think this hurts, at least. We *want* this to be FluencyT or null, so it's probably better to make that explicit.